postman_tests/run_tests.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import absolute_import
+from subprocess import Popen
+import os
+import json
+import sys
+from lib import newman
+from lib import datastore
```

Comment:
I think this lib will reference the one under postman_tests, so should work. Please correct me if im wrong.

Reply:
It will work for now, but since Python 3 defaults to absolute imports, so it can break if some global module `lib` comes into PYTHONPATH first. Better to be safe and use `.lib`